apps/web-giddh/src/app/accounting/journal-voucher/invoice/invoice.component.html

```diff
@@ -0,0 +1,231 @@
+            <div class="clearfix form-inline mrB">
+                <div class="mx-350 form-group" [hidden]="isSalesInvoiceSelected">
+                    <!-- decimalDigitsDirective -->
+                    <label>{{ invoiceNoHeading }}:</label> <input type="text" placeholder="Invoice no."
+                        [(ngModel)]="data.invoiceNumberAgainstVoucher" class="form-control max-height28 appearance-none"
+                        style="max-width:200px" [disabled]="isSalesInvoiceSelected" />
+                </div>
+            </div>
+            <!-- /invoice num -->
+
+
+            <!-- party acc name -->
+            <div class="mx-350 clearfix form-inline mrB">
+                <div class="form-group">
+                    <label>Party A/c Name: <sup>*</sup></label>
+                    <!-- #creditor -->
+                    <input #partyAccNameInputField name="party_name" type="text" placeholder="Party account"
+                        class="form-control max-height28 appearance-none upper-fields"
+                        (input)="onPartyAccFocusInput($event);selectedField='account';focusedField = 'partyAcc';"
+                        (keyup)="searchAccount($event?.target?.value)" (keydown)="detectKey($event)"
+                        [(ngModel)]="creditorAcc.name"
+                        (focus)="onPartyAccFocus($event);selectedField='account';focusedField = 'partyAcc';"
+                        (blur)="onPartyAccBlur();selectedField=null;focusedField = null;filterByText = ''"
+                        style="max-width:200px;" />
+                </div>
+                <!-- [hidden]="!isPurchaseInvoiceSelected" -->
+                <div class="form-group" style="margin-top: 5px;" *ngIf="isPurchaseInvoiceSelected">
+                    <label>Ledger name:<sup>*</sup></label>
+                    <input name="ledger_name" type="text" placeholder="Ledger name"
+                        class="form-control max-height28 appearance-none upper-fields" style="max-width:200px"
+                        (input)="onPartyAccFocusInput($event, 'purchases');selectedField='account';focusedField = 'ledgerName';"
+                        (keyup)="searchAccount($event?.target?.value)" (keydown)="detectKey($event)"
+                        [(ngModel)]="debtorAcc.name"
+                        (focus)="onPartyAccFocus($event,'purchases');selectedField='account';focusedField = 'ledgerName';"
+                        (blur)="onPartyAccBlur();selectedField=null;focusedField = null;filterByText = ''" />
+                </div>
+            </div>
+            <!-- /party acc name -->
+
+            <!--  purchase as INVOICE -->
+            <div class="clearfix mrT1" [style.min-height.vh]="61">
+                <div class="table-responsive bdrT">
+                    <table class="table entry_table">
+                        <thead>
+                            <tr>
+                                <th style="width:70%;">Name of Item</th>
```

Comment:
avoid inline css

Reply:
@rohit-walkover This code is copy-paste and work will be done in near future on this